rules/manager.go

```diff
@@ -176,6 +180,7 @@ func (g *Group) run() {
 		g.Eval(start)
 
 		iterationDuration.Observe(time.Since(start).Seconds())
+		g.EvaluationTimeSeconds = time.Since(start).Seconds()
```

Comment:
This is another race here. When templating we are reading it without a lock.

Add a setter and accessor for it?

Reply:
Done 👍